### PR TITLE
reconnect web socket on sign-in; resolves #1185

### LINKED ISSUE #1185: Chat messages show incorrectly as "guest" if signing in in the middle of chatting

### PR DESCRIPTION
**Pull request by kschzt**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -111,7 +108,7 @@ app.use(methodOverride());
 
 app.use(cookieParser());
 app.use(sessions({
-	cookieName: 'vs050',
+	cookieName: 'vs070',
```
Comment (kkananen) [inline app.js]: could we generate this automatically rather than hard code? (I'm assuming this should be linked to vizor version)

File: app.js [hunk 2]
```diff
@@ -134,12 +131,14 @@ app.use(flash());
 // 	csrf(req, res, next);
 // });
 
-app.use(function(req, res, next)
-{
-	if (!req.user)
+app.upgra
```
Comment (kkananen) [inline app.js]: accidental line?

**Comment by kkananen** [comment]
apart from the extra line, lgtm

would be good to, maybe later, avoid hardcoding 'vs070'

**Comment by kschzt** [comment]
yeah, let's fix vs070 next time we change it

**Pull request merged**